glide.yaml

```diff
@@ -26,9 +26,9 @@ import:
 - package: github.com/go-ole/go-ole
   version: de8695c8edbf8236f30d6e1376e20b198a028d42
 - package: github.com/golang/mock
-  version: bd3c8e81be01eef76d4b503f5e687d2d1354d2d9
+  version: ^1
```

Comment:
Nit: You already mentioned fuzzy versioning consistent with other M3 repos - was just wondering the rationale behind it. Is it so to not have to download multiple versions of what should be compatible versions of the library?

Reply:
I am not sure what you are asking here. As far as I know, only one version of a package is installed with glide. The rationale for this version is just consistency.